pkg/apis/events/test_helper.go

```diff
@@ -0,0 +1,86 @@
+package events
```

Comment:
This is now in our general, released package. My preference would be to move it somewhere away from our production code. Probably in its own package. You might try just changing this line to `package events_test`, but I'm not sure if the other files that import this will be able to.

Reply:
I also think that might be moved to a different package, as pullSubscription conversion test also needs these constants.